payment-example-process-application/kube/manifests/deployment.yaml

```
@@ -20,17 +20,19 @@ spec:
         - containerPort: 8080
           name: http
         env:
-        - name: ZEEBE_CLIENT_BROKER_GATEWAY_ADDRESS
-          value: "camunda-platform-zeebe-gateway.camunda-platform.svc.cluster.local:26500"
-        - name: ZEEBE_CLIENT_SECURITY_PLAINTEXT
-          value: "true"
-        - name: ZEEBE_CLIENT_ID
+        - name: CAMUNDA_CLIENT_MODE
+          value: "self-managed"
+        - name: CAMUNDA_CLIENT_ZEEBE_GRPCADDRESS
+          value: "http://camunda-platform-zeebe-gateway.camunda-platform.svc.cluster.local:26500"
+        - name: CAMUNDA_CLIENT_ZEEBE_RESTADDRESS
+          value: "http://camunda-platform-zeebe-gateway.camunda-platform.svc.cluster.local:8080"
+        - name: CAMUNDA_CLIENT_AUTH_CLIENTID
           value: "payment-app"
-        - name: ZEEBE_AUTHORIZATION_SERVER_URL
+        - name: CAMUNDA_CLIENT_AUTH_ISSUER
           value: http://camunda-platform-keycloak.camunda-platform.svc.cluster.local/auth/realms/camunda-platform/protocol/openid-connect/token
-        - name: ZEEBE_TOKEN_AUDIENCE
+        - name: CAMUNDA_CLIENT_AUTH_AUDIENCE
```

:x: The property name is wrong ([ref](https://docs.camunda.io/docs/apis-tools/spring-zeebe-sdk/getting-started/#self-managed)).

```suggestion
        - name: CAMUNDA_CLIENT_ZEEBE_AUDIENCE
```

---

Thank you @saig0